Fix:

error: implicit declaration of function 'conf_file_set_enable_ipv6' is invalid in C99 [-Werror,-Wimplicit-function-declaration]

https://gitlab.torproject.org/tpo/core/torsocks/-/commit/969d782ad3b560448325ff6e9aa29801d6276a3e
--- src/common/config-file.h
+++ src/common/config-file.h
@@ -109,6 +109,7 @@ int conf_file_set_allow_inbound(const char *val, struct configuration *config);
 int conf_file_set_allow_outbound_localhost(const char *val, struct
 		configuration *config);
 int conf_file_set_isolate_pid(const char *val, struct configuration *config);
+int conf_file_set_enable_ipv6(const char *val, struct configuration *config);
 
 int conf_apply_socks_auth(struct configuration *config);
